Src/WitsmlExplorer.Frontend/components/ContentView.tsx

```diff
@@ -54,7 +55,8 @@ const objectViews: Partial<Record<ObjectType, ReactElement>> = {
 
 const ContentView = (): React.ReactElement => {
   const { navigationState } = useContext(NavigationContext);
-  const { selectedWell, selectedWellbore, selectedLogTypeGroup, selectedLogCurveInfo, selectedObjectGroup, selectedObject, selectedServer, currentSelected } = navigationState;
+  const { selectedWell, selectedWellbore, selectedLogTypeGroup, selectedLogCurveInfo, selectedObjectGroup, selectedObject, selectedServer, currentSelected, displayGraph } =
```

Comment:
`displayGraph` related code can be removed:
* lines 58, 86, 87 ContentView
* lines 21, 43 Routing
* lines 44, 73 navigationContext

Reply:
fixed